tests/test_ImageCollection.py

```diff
@@ -297,6 +297,23 @@ def test_group_interval_with_interval_and_unit(self, jaxa_rainfall):
         assert grouped.size().getInfo() == 3
         assert ee.ImageCollection(grouped.get(0)).size().getInfo() == 24
 
+    def test_group_interval_drop_empty_collections(self, s2_sr):
+        ic = s2_sr.filterDate("2021-01-01", "2021-01-07")
+        grouped: ee.List = ic.geetools.groupInterval(duration=1, unit="day")
+        # Each collection must not be empty
+        for i in range(grouped.size().getInfo()):
+            imgCollection = ee.ImageCollection(grouped.get(i))
+            assert imgCollection.size().getInfo() != 0
+
+    def test_group_interval_without_new_properties(self, s2_sr):
```

Comment:
Running the test is slow and expensive so I try not to be too greedy. I think this one is an overkill, the property will be gone and if it is reintroduce we will see it in other tests.

Reply:
Ok, I deleted that method